xml/cap_depl_stratos.xml

```diff
@@ -249,19 +264,44 @@ NAME            PROVISIONER             AGE
 gp2 (default)   kubernetes.io/aws-ebs   1d
 gp2scoped       kubernetes.io/aws-ebs   1d
 </screen>
+
   <para>
-   Use &helm; to install Stratos:
+   Create a YAML file, called <filename>&stratos-values-file;</filename> in this
+   example, and use it to make configurations to the Stratos &helm; chart.
   </para>
 
+<screen>### example Stratos deployment configuration file
+### &stratos-values-file;
+
+console:
+  # Use local admin user instead of UAA
+  localAdminPassword: <replaceable>changeme</replaceable>
+
+services:
+  loadbalanced: true
+
+kube:
+  storage_class:
+    persistent: gp2scoped
+</screen>
+
   &stratos-tech-preview-note;
 
-<screen>&prompt.user;kubectl create namespace <replaceable>stratos</replaceable>
+  <para>
+   Create a namespace for your Stratos deployment.
+  </para>
+
+<screen>&prompt.user;kubectl create namespace
```

Comment:
Does the "stratos" part in this intentionally appear on a new line?

Reply:
Nope, good catch.